mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1096,7 +1096,7 @@ trait Completions { this: MetalsGlobal =>
       val baseAutoImport: Option[AutoImportPosition] =
         autoImportPosition(pos, text)
       val autoImport: AutoImportPosition = baseAutoImport.getOrElse(
-        AutoImportPosition(lineStart, inferIndent(lineStart, text))
+        AutoImportPosition(lineStart, inferIndent(lineStart, text), false)
```

Comment:
`padTop = false` ?

Reply:
Shoot, forgot that 😬. Will fix.

Reply:
Actually, this seems to be what I intended after looking closer. If I understand this correct, during the completion, if there is no `baseAutoImport` from above, then it puts the import inline like it does in the exhaustive match down around line 1230. So it can be assumed since that's happening that no padding should be added.